common.blocks/radio-group/radio-group.bemhtml

```diff
@@ -27,6 +27,7 @@ block('radio-group')(
                     val : option.val,
                     text : option.text,
                     title : option.title,
+                    tabIndex : option.tabIndex || ctx.tabIndex,
```

Comment:
OMG! looks like very odd case, but since it doesn't take a lot of code complexity maybe we can leave it ;-)